ml-agents/mlagents/trainers/trainer.py

```diff
@@ -17,6 +19,88 @@ class UnityTrainerException(UnityException):
     pass
 
 
+class TrainerMetrics(object):
+    """
+        Helper class to track, write training metrics. Tracks time since object
+        of this class is initialized.
+    """
+
+    def __init__(self, path: str, brain_name: str):
+        """
+        :str path: Fully qualified path where CSV is stored.
+        :str brain_name: Identifier for the Brain which we are training
+        """
+        self.path = path
+        self.brain_name = brain_name
+        self.FIELD_NAMES = ['Brain name', 'Time to update policy',
```

Comment:
Why not make it static ? Put it on line 27 and it will be a static variable common to all of the implementations of this class.